challenges/objects-arrays.js

```diff
@@ -89,37 +110,54 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
-console.log(animalNames);
-
+zooAnimals.forEach(function(zooAnimal)
+{
+  return animalNames.push(`Name: ${zooAnimal.animal_name}, Scientific: ${zooAnimal.scientific_name}`)
+})
+console.log(animalNames)
+;
 /* Request 2: .map()    
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
-const lowerCase = [];
+lowerCase = [];
+lowerCase= zooAnimals.map(function(zooAnimal)
+{
+return zooAnimal.animal_name.toLowerCase();
```

Comment:
Because `.map()` returns a new array without mutating the original, we can refactor this to:
```const lowerCase = zooAnimals.map(zooAnimal => zooAnimal.animal_name.toLowerCase())```